dashboard/src/app/(authenticated)/(diagram)/components/Modals/TaskRun.tsx

```diff
@@ -1,4 +1,5 @@
 import { getVariableValue } from '@/app/utils'
+import { cn } from '@/lib/utils'
```

Comment:
Can you assign a better name to this?

Reply:
We can rename the function, but it's commonly used like this, it's a naming convention shad/cn has, and all their examples include it with the name 'cn' as to reference 'shad/cn'

What do you prepose as this is what everyone in shadcn community uses

Reply:
I see, in that case at least the file name must be different, to reflect what's the scope of that function. `lib/utils` is too generic, this looks like a `classNameHelper` to me.

Reply:
utils is a file that usually starts off with just cn, but over time will have other function helpers like `classNameHelper` you were referring to this would include stuff like `getVariableValue` but the reason it is in that folder structure is because its is a global helper function, the other global helper utils would reside there. Ultimately, it's up to you, if you would still like to change it, what would you propose?